include/mbedtls/ssl_ciphersuites.h

```diff
@@ -405,6 +405,7 @@ struct mbedtls_ssl_ciphersuite_t
 };
 
 const int *mbedtls_ssl_list_ciphersuites( void );
+int mbedtls_ssl_ciphersuites_count( void );
```

Comment:
please change this function to return `const int`

Reply:
Correct, it was more when setting the result as an r value of an assignment to const.
Personally, I prefer puttin const wherever possible, but if it fails your build, keep this as is